src/libs/MoneyRequestUtils.ts

```diff
@@ -21,7 +21,7 @@ function stripSpacesFromAmount(amount: string): string {
  * @param amount - Changed amount from user input
  */
 function addLeadingZero(amount: string): string {
-    return amount === '.' ? '0.' : amount;
+    return amount.startsWith('.') ? `0${amount}` : amount;
```

Comment:
I see some usages of these function.
Wondering in what case it will break, because previously we were just returning `0.`, but now we return `0.amount`

Reply:
Yes sorry, I meant `0amount`

>  technically before we can also return 0amount as amount was "." it would result in "0."

Makes sense 👍